doc/api/fs.markdown

```diff
@@ -122,13 +122,13 @@ Stop watching for changes on the given `fs.FSWatcher`.
 
 ### Event: 'open'
 
-* `fd` {Number} Integer file descriptor used by the ReadStream.
+* `fd` {Integer} Integer file descriptor used by the ReadStream.
 
 Emitted when the ReadStream's file is opened.
 
 ### readStream.path
 
-The path to the file the stream is reading from.
+The path to the file the stream is reading from. Can be a string or a `Buffer`.
```

Comment:
How can I predict this as a user? Do I need to add tests everywhere I want to use this property?